containers/mini

```diff
@@ -0,0 +1,25 @@
+FROM registry.fedoraproject.org/fedora
+
+# Adjust system defaults
+RUN set -x && \
+    sed -i '/tsflags=nodocs/d' /etc/dnf/dnf.conf && \
+    echo "Set disable_coredump false" >> /etc/sudo.conf && \
+    echo 'tester ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# Install necessary packages
+RUN set -x && \
+    dnf install -y tmt && \
+    dnf autoremove -y && \
+    dnf clean all
+
+# Prepare files for experimenting
+RUN useradd --uid 1001 --create-home tester
+COPY ../.fmf /home/tester/.fmf
+COPY ../tests /home/tester/tests
+COPY ../plans/main.fmf /home/tester/plans/main.fmf
+COPY ../plans/features/core.fmf /home/tester/plans/core.fmf
+
+# Run all plans under the regular user by default
+USER 1001
```

Comment:
This won't work for tests where root permissions are required.

Reply:
Ok, let's use root as discussed on the hacking session.